joomla/joomla-cms
# [5.x] Fix `secure` flag for session cookies

### Summary of Changes
The rework of the session management introduced in 5.x broke the code that sets the `secure` flag for the session cookie on sites with enforced HTTPS. The $options variable, containing the config isn't passed to the storage anymore, resulting in the absence of the flag regardless of the configuration state:
https://github.com/joomla/joomla-cms/commit/84776fbe9969ce04ec76ae89a7424474d0b128da#diff-bc8698a8418bcc017e622d13e0d460e94f502f09ae0f9c50cb6cd9f7ede73cb0L90


### Testing Instructions
Enable "force SSL" on a 5.x site, inspect the session cookie using your developer tools. Verify that the flag is absent. Apply the patch and **delete the cookie** in your browser. Refresh.


### Actual result BEFORE applying this Pull Request
No `secure` flag


### Expected result AFTER applying this Pull Request
`secure` flag is set as expected


### Link to documentations
Please select:
